src/javascripts/components/tabs.js

```diff
@@ -33,45 +36,53 @@
       $(id).show().removeAttr('aria-hidden')
     },
 
-    // Accordion mode
-    clickAccordionItem: function (e) {
+    // Tabs mode
+    clickTabItem: function (e) {
       e.preventDefault()
 
+      // Get tab container ID
       var id = $(this).attr('href')
 
-      // Let the main tabs deal with the states
-      $('.app-c-tabs__item a[href="' + id + '"]').click()
+      // Activate current tab
+      GOVUK.tabs.activateCurrentTab(id)
     },
 
     // Close current container on click
     clickCloseContainer: function (e) {
       e.preventDefault()
-      $('.app-c-tabs__item').removeClass('app-c-tabs__item--current')
-      $('.app-c-tabs__heading').removeClass('app-c-tabs__heading--current')
-      $('.app-c-tabs__container').hide().attr('aria-selected', 'true')
+
+      // Get tab container ID and save example object
+      var id = $(this).parent().attr('id')
```

Comment:
The close button doesn't close the container. I think this bit might be getting the wrong `id`.

Reply:
it's missing the '#' from the id when it calls resetTabs